shared-bindings/touchio/TouchIn.c

```diff
@@ -114,18 +114,47 @@ STATIC mp_obj_t touchio_touchin_obj_get_value(mp_obj_t self_in) {
 }
 MP_DEFINE_CONST_FUN_OBJ_1(touchio_touchin_get_value_obj, touchio_touchin_obj_get_value);
 
+
+
 const mp_obj_property_t touchio_touchin_value_obj = {
     .base.type = &mp_type_property,
     .proxy = {(mp_obj_t)&touchio_touchin_get_value_obj,
-              (mp_obj_t)&mp_const_none_obj,
+			  (mp_obj_t)&mp_const_none_obj,
               (mp_obj_t)&mp_const_none_obj},
 };
 
+
+STATIC mp_obj_t touchio_touchin_obj_get_capacitance(mp_obj_t self_in) {
+	touchio_touchin_obj_t *self = MP_OBJ_TO_PTR(self_in);
+	return MP_OBJ_NEW_SMALL_INT(common_hal_touchio_touchin_get_capacitance(self));
+}
+MP_DEFINE_CONST_FUN_OBJ_1(touchio_touchin_get_capacitance_obj, touchio_touchin_obj_get_capacitance);
+
+//|   .. attribute:: capacitance
+//|
+//|     The raw capacitance measurement from the SAMD21 PTC controller using single-capacitance method.
```

Comment:
Please move this comment above the `touchio_touchin_obj_get_capacitance` method to match how value is done.

Also, simplify the comment to "The raw capacitance measurement." without mentioning the SAMD21 because it will be provided by all ports that support touch.